src/augtool.c

```diff
@@ -649,6 +658,24 @@ static void add_transforms(char *ts, size_t tslen) {
     }
 }
 
+static void load_files(char *ts, size_t tslen) {
+    char *command;
+    int r;
+    char *t = NULL;
+
+    while ((t = argz_next(ts, tslen, t))) {
+        r = xasprintf(&command, "load-file %s", t);
+        if (r < 0)
+            fprintf(stderr, "error: Failed to load file %s: could not allocate memory\n", t);
+
+        r = aug_srun(aug, stdout, command);
```

Comment:
Ok. I don't have a strong opinion either way; it's totally fine, I was just curious.